admin/metabox/class-metabox-add-keyword-tab.php

```diff
@@ -20,19 +20,24 @@ public function link() {
 
 		ob_start();
 		?>
-		<li class="wpseo-tab-add-keyword">
-			<button type="button" class="wpseo-add-keyword button">
-				<span aria-hidden="true">+</span>
-				<span class="screen-reader-text"><?php _e( 'Add keyword', 'wordpress-seo' ); ?></span>
-			</button>
-		</li>
+        <li class="wpseo-tab-add-keyword">
+            <button type="button" class="wpseo-add-keyword button">+ <?php _e( 'Add more keywords', 'wordpress-seo' ); ?></button>
```

Comment:
Yoast Premium is using this list-item as well. Users will see 'add more keywords' but when pressing this it will add a single keyword which might be confusing. Is this intended?